### DISCUSSION TITLE
【任务】【黑客松】WTF Academy StarkNet ｜50,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#398] [General]
目前 WTF Solidity 和 WTF Ethers 教程让开发者进入以太坊生态更加容易，但是在Layer 2开发的门槛仍然很高，WTF Academy准备做一个StarkNet上的开发者教学系统，帮助以太坊生态的开发者更顺滑的进入Layer 2开发。

这个项目会参加StarkWare Building黑客松，2月5日结束（特拉维夫，以色列；可以远程）。

**任务描述**：在 WTF Academy 认证系统的基础上，完成WTF Academy StarkNet的MVP（Minimal Viable Product）。包括但不限于以下内容：
1. Cairo教程（中/英文）：最终目的，让开发者完成StarkNet上NFT合约的部署（ERC721）
2. 认证系统产品设计
3. 认证系统UI设计
4. 认证系统前端
5. 认证系统后端
6. StarkNet上的SBT合约，给通过小测的人发放课程认证SBT（ERC1155）。

**赏金**：50,000 $PEOPLE 代币 + 黑客松奖金，根据工作量分配。

请有意参加的WTF贡献者/成员回帖报名，并介绍下自己的技术栈（例如 前端：html/css/js, react），会 Cairo 和 StarkNetjs 是加分项！

如果还没填过WTF社区申请表的，也填一下：[表格](https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform?pli=1)

> WTF Academy 认证系统贡献者自动进入该任务。

参考资料：
1. WTF Solidity开源课程认证系统任务 https://github.com/AmazingAng/WTF-Solidity/discussions/163
2. WTF-Cairo: https://github.com/WTFAcademy/WTF-Cairo
3. StarkNetjs: https://github.com/0xs34n/starknet.js
4. StarkNet Building Hackathon: https://onlydust.notion.site/Projects-b15ff79b608d444f8639ff83c48549a6
5. WTF Academy StarkNet project: https://onlydust.notion.site/WTF-Academy-StarkNet-cec45bfe02694dbc895eb42e8f3f1c31

**Comment by beyond009** [comment]
报名 技术栈前端react全家桶 typescript nodejs 做过evm aptos icp链的项目

**Comment by zhengqian8958** [comment]
这里也报个名。
二十年经验老程序员，啥都会点，希望能帮上忙

**Comment by Lmmmmm** [comment]
报名 技术栈前后端都会点 更擅长前端

**Comment by kongtaoxing** [comment]
报名，我是某211大学学生，会一些starknet.js和cairo 0.10，希望能帮上忙，也能提升自己
